models/models.js

```diff
@@ -30,5 +30,17 @@ exports.selectArticlebyID = (article_id) => {
 }
 
 
+exports.selectComments = (article_id, sort_by = "created_at", order = "desc") => {
+    return db.query(`SELECT * FROM comments 
+        WHERE article_id = $1
+        ORDER BY ${sort_by} ${order};`, [article_id])
+        .then((res) => {
+            if(res.rows.length === 0){
+                return Promise.reject({status:404,msg:'article not found'})
```

Comment:
Just because there are 0 comments, does this mean there is no article? Plan so the code can handle both possibilities